gcloud/datastore/test_batch.py

```diff
@@ -454,3 +443,27 @@ def completed_key(self, new_id):
         new_key = self.__class__(self.dataset_id)
         new_key._id = new_id
         return new_key
+
+
+class _Client(object):
```

Comment:
It was originally lots simpler:  having to grow support for the batch stack etc. makes it more questionable.